tbx/project_styleguide/templates/patterns/molecules/listing/listing--event.html

```diff
@@ -0,0 +1,67 @@
+{% comment %}
+Designed to be included from another template and values passed in.
+
+This component is heavily based on listing--image.html.
+{% endcomment %}
+{% load wagtailimages_tags util_tags %}
+
+{% if listing_image %}
+    {# When updating the image sizes, also update the placeholder-image aspect-ratio & width. #}
+    {% image listing_image fill-485x300 format-webp saturation-0.6 as listing_desktop_image %}
+    {% image listing_image fill-970x600 format-webp saturation-0.6 as listing_desktop_image_retina %}
+    {% image listing_image fill-370x300 format-webp saturation-0.6 as listing_mobile_image %}
+    {% image listing_image fill-740x600 format-webp saturation-0.6 as listing_mobile_image_retina %}
+{% endif %}
+
+<li class="listing-event listing-image listing {{ base_class }}">
+    {% if listing_image %}
+        {# alt text is empty for consistency with listing--avatar.html that includes avatar.html which has empty alt text #}
+        <picture class="listing-image__picture">
+            <source media="(max-width: 598px)" srcset="{{  listing_mobile_image.url }} 1x, {{ listing_mobile_image_retina.url }} 2x" />
+            <source media="(min-width: 599px)" srcset="{{ listing_desktop_image.url }} 1x, {{ listing_desktop_image_retina.url }} 2x" />
+            <img src="{{ listing_desktop_image.url }}" alt="" loading="lazy" class="listing-image__image" />
```

Comment:
can we add the desaturation filter here too

Reply:
It should already be applied a few lines up:
https://github.com/torchbox/torchbox.com/blob/819e5ca7d52293c807d6070dca331bc6c653556c/tbx/project_styleguide/templates/patterns/molecules/listing/listing--event.html#L10-L13

Did you mean something else?

Reply:
Discussed on standup that I can resolve this.